Source/options.cpp

```diff
@@ -579,28 +584,53 @@ string_view OptionCategoryBase::GetDescription() const
 	return _(description.data());
 }
 
+StartUpOptions::StartUpOptions()
+    : OptionCategoryBase("StartUp", N_("Start Up"), N_("Start Up Settings"))
+    , diabloIntro("Diablo Intro", OptionEntryFlags::OnlyDiablo, N_("Intro"), N_("Shown Intro cinematic."), StartUpIntro::Once,
+          {
+              { StartUpIntro::None, N_("None") },
+              // Once is missing, cause we want to hide it from UI-Settings.
+              { StartUpIntro::Always, N_("Always") },
+          })
+    , hellfireIntro("Hellfire Intro", OptionEntryFlags::OnlyHellfire, N_("Intro"), N_("Shown Intro cinematic."), StartUpIntro::Once,
+          {
+              { StartUpIntro::None, N_("None") },

```

Comment:
```suggestion
              { StartUpIntro::None, N_("Never") },
```

Might want to change the enum also

Reply:
No/Yes would also work

Reply:
I changed to OFF/ON, cause we use it already and so there are less text to translate. 🙂